CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Changelog
 
+### 3.0.0-rc.0
+
+* The React component `AppContainer` is now `HotContainer`.
+* The exported Babel plugin is now `react-hot-loader/babel-plugin`.

```

Comment:
Why renaming "babel" to "babel-plugin"? I think it doesn't have so much value except breaking all existing installation of react-hot-loader.

Reply:
https://github.com/gaearon/react-hot-loader/pull/654/commits/dc1daaa61bddf51a2f1fa0a35d533ceadba44055